util/gh-pages/script.js

```diff
@@ -368,4 +374,10 @@ function setTheme(theme, store) {
 }
 
 // loading the theme after the initial load
-setTheme(localStorage.getItem('clippy-lint-list-theme'), false);
+const prefersDark = window.matchMedia("(prefers-color-scheme: dark)");
+const theme = localStorage.getItem('clippy-lint-list-theme');
+if (prefersDark.matches && !theme) {
+    setTheme("coal", false);
```

Comment:
```suggestion
    setTheme("navy", false);
```

Reply:
Navy feels a little too blue to me to be a default dark theme. I feel like the night and day of coal and light is more of what you'd expect when you visit a site offering a dark theme. Also to answer your question, I think coal is the best theme (and rust is definitely my least favorite :smile:).